staging/src/k8s.io/csi-api/pkg/crd/testdata/csinodeinfo.yaml

```diff
@@ -19,6 +19,13 @@ spec:
               driver:
                 description: The CSI driver that this object refers to.
                 type: string
+              installed:
+                description: Whether the CSI driver is installed.
```

Comment:
Is CSINodeInfo not using a spec/status split? This is something I would normally expect to see reported under status

Reply:
I will split it, this object will no longer be backwards compatible which is fine for alpha. But this will need to be documented for the external provisioner which consumes this object